.coafile

```diff
@@ -31,6 +31,9 @@ bears = SpaceConsistencyBear
 bears = GitCommitBear
 shortlog_trailing_period = False
 shortlog_regex = ([^:]*|[^:]+: [A-Z0-9*].*)
+body_close_issue = True
+body_close_issue_full_url = True
+body_close_issue_last_line = True
```

Comment:
Could you add `body_enforce_issue_reference = False` explicitly? We shouldn't rely on bear defaults in this case, as this is definitely the setting we want for coala ;) Better explicit than implicit :)